testing/maas_billing_tests_independent/tests/test_tokens.py

```
@@ -0,0 +1,51 @@
+from conftest import bearer, parse_usage_headers, USAGE_HEADERS, ensure_free_key, ensure_premium_key
+import os, json, base64
+
+def _b64url_decode(s):
+    pad = "=" * (-len(s) % 4)
+    return base64.urlsafe_b64decode((s + pad).encode("utf-8"))
+
+def test_minted_token_is_jwt(http, base_url, maas_key):
```

Ruff (0.13.2)

Unused function argument: `http`

(ARG001)